### PR TITLE
Add test for Functions::FEFieldFunction::compute_point_locations

### PR DESCRIPTION
**Pull request by GivAlz**
Since I couldn't find one, I'm adding a test for Functions::FEFieldFunction::compute_point_locations
This is an important base line since I shall make another PR with a new faster version of this function (and some other modifications, such as avoiding using a dummy FEFieldFunction altogether...but that's for another PR).
P.S. If I missed the test altogether please just tell me where it is, but I really couldn't find it!

### CODE REVIEW

File: tests/fe/fe_compute_point_locations_01.cc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  dof_handler.distribute_dofs(fe);
+
+  // Creating the random points
+  std::vector<Point<dim>> points;
+
+  for (size_t i=0; i<n_points; ++i)
+    {
+      Point<dim> p;
+      for (unsigned int d=0; d<dim; ++d)
+        p[d] = double(std::rand())/RAND_MAX; //Normalizing the value
```
Comment (tjhei) [inline tests/fe/fe_compute_point_locations_01.cc]: can you use ``Testing::rand()`` instead and remove the ``srand`` call above?

File: tests/fe/fe_compute_point_locations_01.cc [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  std::vector<std::vector<unsigned int> > maps;
+
+  // Creating a dummy vector/fe_field_function in order to use FEFieldFunction
+  Vector<double> dummy;
+  Functions::FEFieldFunction<dim> fe_function(dof_handler, dummy, StaticMappingQ1<dim,dim>::mapping);
+  size_t n_cells = fe_function.compute_point_locations(points, cells, qpoints, maps);
+
+  deallog << "Points found in " << n_cells << " cells" << std::endl;
+
+  // Testint if the transformation is correct:
```
Comment (tjhei) [inline tests/fe/fe_compute_point_locations_01.cc]: typo "testing"

File: tests/fe/fe_compute_point_locations_01.cc (hunk 1 above)
Comment (GivAlz) [reply tests/fe/fe_compute_point_locations_01.cc]: Ok, just changed this call and the typo afterwards...thank you!

**Comment by luca-heltai** [comment]
@GivAlz , you changed the std::rand function, but did not update the output of the tests... please update the output file.

**Comment by GivAlz** [comment]
Sorry... yesterday I was too tired and didn;t check properly...now it works

**Pull request merged**